hal/src/main/native/athena/HAL.cpp

```diff
@@ -287,6 +292,68 @@ size_t HAL_GetSerialNumber(char* buffer, size_t size) {
   }
 }
 
+void InitializeRoboRioComments(void) {
+  if (!roboRioCommentsStringInitialized) {
+    std::error_code ec;
+    std::unique_ptr<wpi::MemoryBuffer> fileBuffer =
+        wpi::MemoryBuffer::GetFile("/etc/machine-info", ec);
+
+    std::string_view fileContents;
+    if (fileBuffer && !ec) {
+      fileContents =
+          std::string_view(reinterpret_cast<const char*>(fileBuffer->begin()),
+                           fileBuffer->size());
+    } else {
+      return;
```

Comment:
This looks incorrect?

Reply:
Yep... early return from debugging. Thanks!